### PR TITLE
Updates Hypospray belt and chem colours.

### PR DESCRIPTION
**Pull request by Arctrooper8**
## About The Pull Request

New hypospray belt:
image
order of items (bottle means a bottle of the previously listed chem): Big Bicard, Bottle, Big Kelo, Bottle, Big Trama, bottle, Big Tricord, bottle, Big dylo, Big Ina, Big Dex, Big Space, Big imialky (a mixture hypo that displays how multiple chems in one hypo works) big QC, normal hypervene, health analyzer.

New colours:
QC: (pink because it restores blood and halts IB)
image
Nanoblood: (red for blood)
image
Oxy: (Black because it's a stronger painkiller)
image
Para: (Light grey because it's a weaker painkiller)
image
RR: (Dark Red because it has red in the name and it's a dangerous drug)
image

Also updates all hypospray descriptions to explain what they do on examine.

## Why It's Good For The Game

Hypospray belt is a bit of a joke right now, it's clunky to use, lacks most of the chems you'd need, and simply pales in comparison to the other belts.

By changing the chem liquid colors to match pill bottles you can have a better visual indicator of what each hypospray is, useful when they move around so much.
Using big hyposprays also 

## Changelog
:cl:
expansion: Changes a few chem colours, nanoblood, QC, painkillers, and RR.
del: Fully removes Bottlecases.
qol: Chemical liquid colours now match their pill bottles.
qol: Premade Hypospray descriptions have been updated to explain what their chems do.
balance: Changes the content of Hypospray belt.
fix: No more invisible things in hypospray belt.
/:cl:

### CODE REVIEW

**Comment by goose4429** [comment]
Why are you removing bottle cases?

**Review by Kuro020 — APPROVED** [review]

**Comment by Arctrooper8** [comment]
> Why are you removing bottle cases? I'm using it without regards to your hypo belt. Please don't remove things 'just because' - people asked me what happened to injector case like 10 times already. Now you're removing bottle cases because... why? You want to break corpsman vendor and people loadouts again?

How are you even getting bottle cases without the hypospray belt? They're only used in the hypo belt. If you're talking about syringe cases (which do the exact same thing) those are still there.

> Also, giving 10 big hypos is an overkill. Although I feel that it's a cool thing to do to make it more useful. Still, 10 too many - its 80 corpsman points.

Big hypos are overpriced for what they give, they're twice as big as a normal hypo yet cost 4 times. I feel a corpsmen only medical belt that can only hold hypos, injectors, beakers, and bottles having a bunch of big hypos isn't that absurd. Especially when you see how much pill bottles can hold.

> Also, the problem with belt was low chem capacity. Normal belt has max 21x240u chems, this PR in current state has 12x(120u+60u) = 6x180u. That's still super meh. That is also the reason why cases are a thing.

Exactly why I gave them Big hyposprays, the bottle cases were a clunky solution and somewhat questionable solution. 

> **As for fixes, fix good. Sprites good. Other than that, in current state PR bad. You should split balance changes and sprite changes into different PRs. Atomize.**

If worst comes to worst I'll split them, but I feel they're related enough that I can justifty them being in the same PR.

Overall, this probably isn't gonna make hypospray belt used more. But it will make it less painful for those trying it or doing a gimmick.

**Comment by goose4429** [comment]
> How are you even getting bottle cases without the hypospray belt? They're only used in the hypo belt. If you're talking about syringe cases (which do the exact same thing) those are still there.

Huh, okay then. But still, have problem - see below.

> Big hypos are overpriced for what they give, they're twice as big as a normal hypo yet cost 4 times. I feel a corpsmen only medical belt that can only hold hypos, injectors, beakers, and bottles having a bunch of big hypos isn't that absurd. Especially when you see how much pill bottles can hold.

I think making big hypo cost 8 or so would be reasonable, probs will do soon. But anyway, there's a high chance that maints will ask to rebalance just because of this. I'd make non-relevant hypos (QC or whatever) smaller just to save you nerves. 

> Exactly why I gave them Big hyposprays, the bottle cases were a clunky solution and somewhat questionable solution.

But you've decreased total chem capacity. That's what I am saying. It's possible to overcome it easily just by taking pill bottles and using them to refill hypos, but that doesn't help the belt much. I'd just stick with both big hypos and big cases. I don't really get why you'd remove them, mostly because leftclick bottle leftclick hypo is same as rightclick case leftclick hypo, and latter has more chems.

> Overall, this probably isn't gonna make hypospray belt used more. But it will make it less painful for those trying it or doing a gimmick.

Hope so. Although chem capacity for base meds is _really_ low. I'm rooting for cases to be intact, so that belt is actually much better.

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 1]
```diff
@@ -285,180 +285,204 @@
...
 
 
+/obj/item/reagent_containers/hypospray/advanced/update_icon()
+	. = ..()
 
-/obj/item/reagent_containers/hypospray/advanced/tricordrazine
-	list_reagents = list(/datum/reagent/medicine/tricordrazine = 60)
+	overlays.Cut()
+	if(reagents.total_volume)
+		var/image/filling = image('icons/obj/reagentfillings.dmi', src, "[icon_state]10")
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/hypospray.dm:293]: use update overlays

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 2]
```diff
@@ -285,180 +285,204 @@
...
 
+		var/percent = round((reagents.total_volume / volume) * 100)
+		switch(percent)
+			if(0 to 9)			filling.icon_state = "[initial(icon_state)]-10"
+			if(10 to 24) 		filling.icon_state = "[initial(icon_state)]10"
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/hypospray.dm:298]: Dont use same line ifs

File: code/game/objects/items/reagent_containers/hypospray.dm [hunk 3]
```diff
@@ -285,180 +285,204 @@
...
-	desc = "A hypospray loaded with several doses of advanced healing and painkilling chemicals. Intended for use in active combat."
+/obj/item/reagent_containers/hypospray/advanced/examine(mob/user as mob)
+	. = ..()
+	if(get_dist(user,src) > 2)
+		. += span_warning("You're too far away to see [src]'s reagent display!")
+		return
+
+	. += display_reagents(user)
+
+/obj/item/reagent_containers/hypospray/advanced/proc/display_reagents(mob/user)
```
Comment (TiviPlus) [inline code/game/objects/items/reagent_containers/hypospray.dm:316]: Autodoc

**Review by TiviPlus — CHANGES_REQUESTED** [review]
As a side note I hate reviewing balance prs and you not atomizing that out is why it took me this long

**Pull request merged**